dashboard/views.py

```diff
@@ -263,8 +262,6 @@ def dispatch(
 def resubmit_pset(request: HttpRequest, pk: int) -> HttpResponse:
     pset = get_object_or_404(PSet, pk=pk)
     student = pset.student
-    if student.semester.active is False:
```

Comment:
can we verify the logged in user is active for the _current_ semester of OTIS instead? properly speaking random alums should not be able to resubmit their old psets for me to review